Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.GPU.sb.in

```diff
@@ -988,3 +988,5 @@
         (extension-class "com.apple.mediaserverd.read")
         (apply subpath issue-extension-secondary-paths)))
 #endif
+
+#include <WebKitAdditions/com.apple.WebKit.GPU.sb.in>
```

Comment:
Don't we need a check to avoid doing this with the OpenSource SDK?

Reply:
Ah, good catch, fixing now!

Thanks for reviewing!